src/System.Data.SqlClient/src/Configurations.props

```diff
@@ -12,6 +12,7 @@
     <BuildConfigurations>
       $(PackageConfigurations)
       uap-Windows_NT;
+      netcoreapp;
```

Comment:
@weshaggard do we need to move netcoreapp to package config instead of keeping it in build config?

Reply:
Yes you will want to ship it in the package so it should be in the package config